data/dataset/redis_dataset.yml

```diff
@@ -0,0 +1,119 @@
+dataset:
+  - fides_key: fidesops_redis_dataset
+    name: Fidesops Redis Dataset
+    description: Fields stored in temporary storage to support Fidesops request execution
+    collections:
+      - name: hash_table
+        fields:
+          - name: EN_ACCESS_GRAPH__<privacy_request_id> # Usage: This graph is summarized and sent to Fideslog to create high level insight into how graphs change between retries to inform future features.
```

Comment:
I use <> here to denote dynamic keys. It highlights another example of a scenario we don't currently handle in fidesops where data might have dynamic keys.

In MongoDB, for example, we can currently only perform access requests and erasure requests on data with pre-determined definitive paths, because the current design expects you to be able to build a named path to the given data.

Example 1: Fidesops can access and mask this data.
```json
{
   "restaurants":[
      {
         "name":"Fred's Burgers",
         "type":"American",
         "rating":4
      },
      {
         "name":"Roma's",
         "type":"Italian",
         "rating":3
      }
   ]
}
```

Example 2: Fidesops currently cannot access data nested under restaurants because the keys change.

```json
{
   "restaurants":{
      "Fred's burgers":{
         "type":"American",
         "rating":4
      },
      "Roma's":{
         "type":"Italian",
         "rating":3
      }
   }
}
```